client/verta/tests/test_artifacts.py

```diff
@@ -434,6 +434,37 @@ def predict(self, data):
         assert experiment_run.get_model().__dict__ == custom.__dict__
         assert experiment_run.get_model().predict(strs) == custom.predict(strs)
 
+    def test_pyspark(self, experiment_run, in_tempdir):
```

Comment:
I suspect it's going to be difficult to get this running nightly (due to requiring Spark).

Reply:
We can install in the container maybe =)